Dockerfile

```diff
@@ -84,6 +84,7 @@ RUN wget -q -O /usr/local/bin/local-php-security-checker https://github.com/fabp
 
 # Apache config.
 COPY ./files/apache2.conf /etc/apache2/apache2.conf
+COPY ./files/mpm_prefork.conf /etc/apache2/mods-available/mpm_prefork.conf
```

Comment:
mods-enabled?

Reply:
Added an `a2enmod` instead to follow standards.